src/TriArithmetic/conversions.js

```diff
@@ -21,3 +21,36 @@ export function wordToDec(word) {
   }
   return dec;
 }
+
+/**
+ * Converts a dec to balanced ternary representation
+ * @param {decimal} decimal
```

Comment:
@param {number} decimal number to be converted
@returns {Tri[]} the array of tris giving the balanced ternary representation of the input value